docs/testing.md

```diff
@@ -3,8 +3,8 @@ Testing instructions
 Tests can be found under drupal/tests folder.
 Test are run using Codeception.
 
-To get started follow these steps:
-1. Uncomment or add selenium role in environment(s) that you will run tests on:
+To get started, follow these steps:
+1. Uncomment or add selenium role in environment(-s) that you will run tests on:
```

Comment:
No need for that hyphen in English.
http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Plurals.html?old=Plurals11.html
http://www.grammarphobia.com/blog/2015/04/parenthetical-plurals.html